packages/xod-client/src/user/actions.js

```diff
@@ -0,0 +1,10 @@
+import { UPDATE_COMPILE_LIMIT } from './actionTypes';
+
+export const updateCompileLimit = COMPILE_HOSTNAME => () => dispatch =>
+  fetch(`${COMPILE_HOSTNAME}/limits`)
```

Comment:
Still no error handling and no retries...

Reply:
@brusherru No error handling is required for this call.

Reply:
@brusherru Should the retries be the responsibility of this component as opposed to the app?

Reply:
But no one handles errors. So if something went wrong and `fetch` will reject — it will be unhandled rejection... and there will be no reaction for the action

Reply:
About retries...
In case that this function fetches data and then updates state — I think it could do retries too.
Not unlimited retries, of course.
But app should call this function later if the state still contains `null` limits. Maybe when a user clicks on `upload` menu item.
IMHO.
What do you think?